rasa_nlu/train.py

```diff
@@ -73,11 +73,11 @@ def do_train(config, component_builder=None):
     training_data = load_data(config['data'])
     interpreter = trainer.train(training_data)
     persisted_path = trainer.persist(config['path'], persistor, model_name=config['name'])
-    return trainer, interpreter, persisted_path
+    # return trainer, interpreter, persisted_path
```

Comment:
Convert these objects to dicts and write some classes' methods to instantiate them back